docker_images/diffusers/tests/test_api.py

```diff
@@ -8,7 +8,10 @@
 # Must contain at least one example of each implemented pipeline
 # Tests do not check the actual values of the model output, so small dummy
 # models are recommended for faster tests.
-TESTABLE_MODELS: Dict[str, str] = {"text-to-image": "CompVis/ldm-text2im-large-256"}
+TESTABLE_MODELS: Dict[str, str] = {
+    "text-to-image": "hf-internal-testing/tiny-stable-diffusion-pipe",
+    "image-to-image": "hf-internal-testing/tiny-controlnet",
```

Comment:
Would be great to add a second test for `image-to-image` that is not controlNet. An example of testing multiple models can be seen here, but it will need some slight changes in the tests https://github.com/huggingface/api-inference-community/blob/main/docker_images/speechbrain/tests/test_api.py#L11

Reply:
Done